_about_pages/definitions.adoc

```diff
@@ -0,0 +1,177 @@
+scheduled in advance of a link:#Conference[Conference], or proposed by
+any participant during the course of a Conference, and held when time
+permits. BOFs are typically scheduled for 30 minutes to no more than an
+hour.
+
+BOFs are generally interactive discussions on a problem, an issue, or a
+potential work area. A BOF may (but need not) begin with or be based on
+some form of presentation offered by the person requesting the BOF. A
+BOF is a good way to initiate discussion in an area and determine
+whether there is enough interest to try and form a link:#PC[Provisional
+Committee] to explore the issue further. A BOF may result in the
+formation of an Provisional Committee or other follow-on activity such
+as a link:#Workshop[Workshop].
+
+[[Conference]]
+=== CalConnect Conference
+
+CalConnect holds two or three CalConnect Conferences (sometimes just
+"Conferences") each year.  Conferences are currently 4 days, either
+Monday-Thursday or Tuesday-Friday.  The Conference is are open to all
+member representatives; non-members may attend as
+link:#Observer[Observers]. In some cases a Conference may also involve a
+public link:#Workshop[Workshop] which will also allow any non-member to
+participate upon registration. +
+ +
+CalConnect Conferences are where link:#PC[Provisional Committees] report
+on their findings and where link:#TC[Technical Committees] report on
+their work, hold discussions and resolve outstanding issues. Conferences
+also provide an opportunity to foster new work and ideas through BOFs
+and Workshops, and a general plenary session. See
+http://calconnect.org/conferences.shtml[CalConnect Conferences] for more
+information. +
+
+[[Consortium]]
+=== Consortium (The Calendaring and Scheduling Consortium)
+
+The Calendaring and Scheduling Consortium is the full name of
+CalConnect; it is incorporated in California ias a 501(c)6 Non-Profit
+Corporation. CalConnect is a Service Mark of the Consortium. Generally
+the organization is simply referred to as "CalConnect". +
+
+=== Interoperability Test Event
+
+As of 2019 CalConnect is not holding separate Interoperability Test
+Events.  Testing opportunities are integrated as working sessions into
+the Conferences or the day preceeding or following a Conference.  If the
+demand for testing increases we will look into separate testing
+opportunities once again. +
+
+=== Member
+
+A http://membership[Member] of CalConnect is normally an organization
+(e.g. vendor, company, governmental organization, academic institution)
+except in the case of an Individual Member. Member Representatives are
+individuals from a member who represent their organization in CalConnect
+activities. Each member has a Primary Representative who is responsible
+for the member's involvement in CalConnect, and who votes on behalf of
+the member when necessary.
+
+=== Observer
+
+An http://events/events-activities/observers[Observer] is an individual
```

Comment:
These links have to be fixed to `link:/events/events-activities/observers[Observer]`.